kuiper-game/Systems/Events/IEvent.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Kuiper.Systems.Events
+{   
+    public interface IEvent
+    {
+        DateTime EventTime { get; }
+        string EventName { get; }
+
+        void Execute(string[] args);
```

Comment:
All events inherit from this interface and a simple execute method is standard for all. Currently takes some arguments, but they might not be needed (or have anywhere they come from...)